spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/ConcatenationRuleInterpreter.scala

```diff
@@ -22,9 +22,10 @@ import za.co.absa.enceladus.conformance.ConfCmdConfig
 import za.co.absa.enceladus.conformance.interpreter.{ExplosionState, RuleValidators}
 import za.co.absa.enceladus.dao.MenasDAO
 import za.co.absa.enceladus.model.conformanceRule.{ConcatenationConformanceRule, ConformanceRule}
-import za.co.absa.enceladus.utils.transformations.DeepArrayTransformations
 
 case class ConcatenationRuleInterpreter(rule: ConcatenationConformanceRule) extends RuleInterpreter {
+  import za.co.absa.spark.hats.Extensions._
```

Comment:
To discuss: Based on the repeated import occurrence, I wonder, if it would be possible and useful to extract the logic using `df.nestedWithColumnExtended' into one code block in the ancestor?

Reply:
I'm not sure. `df.nestedYYYY()` are meant to extend Dataframe DSL and to have a feel of being part of its API. I'd like them not to be grouped as something special.

Reply:
Fair enough.